spring-cloud-app-broker-deployer-cloudfoundry/src/main/java/org/springframework/cloud/appbroker/deployer/cloudfoundry/CloudFoundryAppDeployer.java

```diff
@@ -767,6 +767,7 @@ private Mono<String> createSpace(String spaceName) {
 						})
 						.doOnError(e -> LOG.error(String.format("Error creating space. spaceName=%s, " +
 							ERROR_LOG_TEMPLATE, spaceName, e.getMessage()), e))
+						.onErrorResume(e -> Mono.empty())
```

Comment:
Wound't this make the scenario to fail silently?

Reply:
The opposite, when the two threads try to create the same space it typically fails, now we can assume that when the space exists, we can carry on with the flow.